iXBRLViewerPlugin/viewer/src/js/report.js

```diff
@@ -38,6 +38,10 @@ export class XBRLReport {
         return this.reportSet.factsForReport(this);
     }
 
+    targetDocument() {
+        return this._reportData.target ?? undefined;
```

Comment:
Out of curiosity, why `?? undefined`?  Is that normalizing `null`->`undefined` and also `undefined` for older serialized viewers?  Or is there a broader convention of using `undefined` for "explicitly assigned to nothing" in JavaScript or this repo?

Reply:
Yes, it's to normalize the JSON `null` from new document and absence of property in older viewers.

I don't think we're very consistent on `null` vs `undefined`, but I agree that `null` makes more sense here (now changed).